### DISCUSSION TITLE
Time Lapse Photos - How to add a TIMESTAMP

### DISCUSSION THREAD

**Discussion by Kim-Ocean** [gopro/labs#368] [Q&A]
Hello dear Forum Members,

maybe someone can help me answer my question:

I use a Hero 11 in 2s photo timelapse mode. Now I would also like to add a timestamp to each picture. Could you tell me how I can do so? 

Thanks for your help!

**Comment by dnewman-gpsw** [comment]
This is not supported with Labs.  You will need to use a batch tool in post.
